src/modules/fancyzones/editor/FancyZonesEditor/GlobalSuppressions.cs

```diff
@@ -25,14 +25,14 @@
 
 [assembly: SuppressMessage("Microsoft.Design", "CA1009:DeclareEventHandlersCorrectly", Scope = "member", Target = "Microsoft.Templates.Core.Locations.TemplatesSynchronization.#SyncStatusChanged", Justification = "Using an Action<object, SyncStatusEventArgs> does not allow the required notation")]
 
-// Non general supressions
+// Non general suppressions
```

Comment:
This file should not be changed since is generated by a tool that is not part of this repository.

Reply:
Can you help me get the tool fixed? Is there a repository for it, or is it just something you can file a bug against? (I'll obviously drop this, but my goal is to cascade fixes along until they're gone...)